tools/jit/gen_jit_dispatch.py

```diff
@@ -369,14 +369,15 @@ def declkey(decl):
         return [sorted(g, key=declkey) for g in grouped_decls]
 
     # We need to add methods implemented manually in TensorImpl
+    # TODO: This seems to claim sizes() returns an int64_t.  Really?
```

Comment:
our `Tensor::sizes()` does return a int64_t iirc. It's number of dimensions. And yes I agree it's confusing :P